src/server/data/database.js

```diff
@@ -163,17 +163,52 @@ export function removeBoard(id) {
 }
 
 export function getCardLists(boardId) {
-  return cardLists.filter((list) => list.boardId == boardId);
+  return cardLists.filter((list) => list.boardId === boardId);
 }
 
 export function getCardList(id) {
-  return cardLists.filter((list) => list.id == id)[0];
+  return cardLists.filter((list) => list.id === id)[0];
 }
 
+// Ordered by cardListRank
 export function getCards(cardListId) {
-  return cards.filter(card => card.cardListId === cardListId);
+  let c = cards.filter(card => card.cardListId === cardListId);
+  c = c.sort((a, b) => {
```

Comment:
Agreed, i actually don't think we need to sort on the client side right now since it comes back sorted from the server.